iep-spring-leader-redis-cluster/README.md

```diff
@@ -0,0 +1,19 @@
+
+## Description
+
+Redis Cluster backed `LeaderDatabase` implementation.
+
+### Usage
+This `LeaderDatabase` implementation can be instantiated directly, injected programmatically, or
+injected via Spring.
+ 
+Once an instance is obtained, `initialize()` must be called. This is not called by default, since it
```

Comment:
It is typically called automatically by the leader elector this will get injected into.